x/nexus/keeper/migrate.go

```diff
@@ -4,10 +4,25 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// GetMigrationHandler returns the handler that performs in-place store migrations from v0.14 to v0.15. The
+// GetMigrationHandler returns the handler that performs in-place store migrations from v0.24 to v0.25. The
 // migration includes:
+//   - migrate maintainer states
 func GetMigrationHandler(k Keeper) func(ctx sdk.Context) error {
 	return func(ctx sdk.Context) error {
+		migrateMaintainerStates(ctx, k)
+
 		return nil
 	}
 }
+
+func migrateMaintainerStates(ctx sdk.Context, k Keeper) {
+	for _, chainState := range k.getChainStates(ctx) {
+		for _, maintainerState := range chainState.MaintainerStates {
+			maintainerState.Chain = chainState.ChainName()
+			k.setChainMaintainerState(ctx, &maintainerState)
+		}
+
+		chainState.MaintainerStates = nil
+		k.setChainState(ctx, chainState)
```

Comment:
The `Maintainers` field was set as reserved and so isn't populated on read, and when writing it's not included anymore so it should not be in storage anymore after the migration, correct?

Reply:
that's right. reserved means gone.